pages/contribute.md

```diff
@@ -0,0 +1,114 @@
+
+### MDAKits Registry
+The [MDAKits Registry]({{ site.mdakits.registry }}) lists active projects contributed by the community. These projects cover various applications, such as enhanced trajectory analysis, molecular simulations, and integration with machine learning frameworks.
+
+### How to Get Started
+- Explore existing MDAKits in the [MDAKits Registry]({{ site.mdakits.registry }}) to find projects that match your interests. Each entry includes functionality, installation instructions, and contribution guidelines.
+- Find open issues in [MDAKits GitHub Issues]({{ site.mdakits.issues }}).
+- Propose a new MDAKit by following the [MDAKits Guide]({{ site.mdakits.guide }}).
+- Contribute to an existing MDAKit by collaborating with maintainers to improve functionality, fix bugs, or add new features.
+
+MDAKits follow MDAnalysis coding standards but operate independently. Contributions to these projects are highly encouraged and help expand the MDAnalysis ecosystem.
+
+## Contributing to the Documentation
+Good documentation is essential to making MDAnalysis accessible to users and developers. MDAnalysis maintains two types of documentation: the [User Guide]({{ site.docs.userguide.url }}) and the [API Reference]({{ site.docs.mdanalysis.url }}/stable/index.html).
+
+### User Guide
+The [User Guide]({{ site.docs.userguide.url }}/stable/index.html) provides tutorials and explanatory content to help users install and learn MDAnalysis. You can contribute by updating unclear explanations, adding new tutorials or examples, or fixing outdated information. 
+
+### How to Get Started
+- Work on the [User Guide Repository]({{ site.docs.userguide.repo }}).
+- Follow the [Contributing to the User Guide]({{ site.docs.userguide.url }}/stable/contributing_docs.html).
+- Check for open documentation-related issues in the [User Guide GitHub Issues]({{ site.docs.userguide.issues }}).
+
+### API Reference
+The [API reference]({{ site.docs.mdanalysis.url }}/stable/index.html) is **automatically generated** from *docstrings* in the codebase. It provides a structured reference for developers working with MDAnalysis. You can contribute by improving *docstrings* for functions, classes, and modules or updating outdated documentation.
+
+To get started:
+- Work on the [MDAnalysis Main Code Repository]({{ site.github.repo }}).
+- Follow the [API Documentation Contribution Guide]({{ site.docs.userguide.url }}/stable/contributing_code.html#working-with-mdanalysis-docs).
+- Browse open API documentation issues in [GitHub Issues]({{ site.github.issues }}).
+
+## Mentoring & Outreach Programs
+MDAnalysis actively participates in various mentoring and outreach programs **to help students, researchers, and early-career contributors gain experience in open-source development and scientific software**. Through these programs, participants receive mentorship, contribute to real-world projects, and develop valuable skills in computational science and software engineering.
```

Comment:
```suggestion
MDAnalysis has actively participated in various mentoring and outreach programs **to help students, researchers, and early-career contributors gain experience in open-source development and scientific software**. Through these programs, participants receive mentorship, contribute to real-world projects, and develop valuable skills in computational science and software engineering.
```

Reply:
changing this to past tense - as for this year at least we are not taking on any other commitments